SQLite.swift.podspec

```diff
@@ -1,17 +1,32 @@
-$LOAD_PATH << '.'
-require 'Supporting Files/podspec.rb'
+#
+# `pod lib lint SQLite.swift.podspec' fails - see
```

Comment:
Are you doing "pod lib lint ..." perhaps?  You need to use "pod spec lint ...", per https://github.com/svdo/swift-netutils/issues/4 which leads to https://github.com/CocoaPods/CocoaPods/issues/4607. See comment on line 2. That same error cost me several hours until I found the pre-existing issue :(

I agree that the deployment target is a good idea regardless though. (Though to be honest, I'm not sure it's possible to have Swift on < 8.0 anyway, so I think this comes out in the wash).